Update window_functions.md (#4155)
duckdb/duckdb-web: docs/sql/functions/window_functions.md

Framing specifies a set of rows relative to each row where the function is evaluated.
The distance from the current row is given as an expression either `PRECEDING` or `FOLLOWING` the current row.
This distance can either be specified as an integral number of `ROWS`
or as a `RANGE` delta expression.
For a `RANGE` specification, there must  be only one ordering expression,
and it has to support addition and subtraction (i.e., numbers or `INTERVAL`s).

> The default frame is from `UNBOUNDED PRECEDING` to `UNBOUNDED FOLLOWING` for all window functions, except it is `UNBOUNDED PRECEDING` to `CURRENT ROW` when an `ORDER BY` clause is present. 

Author: ```suggestion
The default frame is from `UNBOUNDED PRECEDING` to `UNBOUNDED FOLLOWING` when no `ORDER BY` clause is present and from `UNBOUNDED PRECEDING` to `CURRENT ROW` when an `ORDER BY` clause is present. 
```

> The default frame is from `UNBOUNDED PRECEDING` to `UNBOUNDED FOLLOWING` for all window functions, except it is `UNBOUNDED PRECEDING` to `CURRENT ROW` when an `ORDER BY` clause is present. If the `OVER` specification contains an `ORDER BY` clause or from `UNBOUNDED PRECEDING TO UNBOUNDED FOLLOWING` otherwise. 

Author: ```suggestion
The default frame is from `UNBOUNDED PRECEDING` to `UNBOUNDED FOLLOWING` for all window functions, except it is `UNBOUNDED PRECEDING` to `CURRENT ROW` when an `ORDER BY` clause is present. 
```
